packages/oidc-dpop-client-browser/package.json

```diff
@@ -13,7 +13,8 @@
     "clean": "npm run clean-module && npm run clean-browser && npm run clean-examples && rm -rf ./coverage",
     "clean-module": "rm -rf ./dist",
     "clean-browser": "rm -rf ./browserDist",
-    "build": "rollup --config rollup.config.js",
+    "rollup": "rollup --config rollup.config.js",
```

Comment:
Maybe we can just remove rollup altogether?

Reply:
I removed tsc, and kept rollup, since you found the config issue.